src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -74,6 +78,17 @@ static void checkArg(string arg, string name)
                 WorkingDirectory = workingDir,
                 GuiApp = guiApp
             };
+
+            // create iteration set up process helper
+            logger.Log($"Iteration set up: {iterationSetup} (args: {setupArgs})");
+            ProcessHelper setUpProcHelper = null;
+            if (!String.IsNullOrEmpty(iterationSetup))
+            {
+                setUpProcHelper = SetupIteration(iterationSetup, setupArgs, workingDir);
+            }
+
+            // TODO: create clean up proc helper
```

Comment:
Are you going to do this now? If not let's remove the TODO and we'll just know we need to come back to this if we ever have a scenario that warrants it.